### ISSUE TITLE
Email improvements

### ISSUE THREAD

**Issue by robert7** [robert7/nixnote2#23]
Moved here from https://github.com/robert7/nixnote2/issues/18
> I just sent another pull request. I made the "email note" function
> relative to each note window, instead of app global. It still works
> globally, but now also works from every note window. Everything users
> might have done before still works, yet there's now new functionality
> from each note window.
> This is actually a cleaner design anyway since the email function
> applies to a particular note, not to the application itself.

**Comment by robert7** [comment]
yes. of course. I'll review it soon
I just wanted to open a ticket regarding "email changes" .. where I did put together your changes and my plans :)

**Comment by MRC01** [comment]
Suggestion: add this capability to use a separate email client to send email, yet keep the current code. While most Linux distros have an email client, people who don't use it and haven't configured it, will need the existing Nixnote code, despite its limitations.

**Comment by robert7** [comment]
Yes. I will leave the existing build client as option (as people may already using it).

**Comment by robert7** [comment]
@MRC01: your changes are merged - thx!
I also created a "develop" branch. So next PRs should probable be directed to this branch.
There are minor changed to the build script `build-with-qmake.sh`, but you probably already noticed.
